rc/tools/doc.kak

```diff
@@ -137,9 +137,19 @@ define-command -params 1 -hidden doc-render %{
 define-command -params 1..2 \
     -shell-script-candidates %{
         if [ "$kak_token_to_complete" -eq 0 ]; then
-            find "${kak_runtime}/doc/" -type f -name "*.asciidoc" | sed 's,.*/,,; s/\.[^/]*$//'
+            (
+                find "${kak_runtime}/doc/" -type f -name "*.asciidoc"
+                find "${kak_runtime}/rc/" -type f -name "*.asciidoc"
+                find "${kak_config}/autoload/" -type f -name "*.asciidoc"
+            ) | sed 's,.*/,,; s/\.[^/]*$//'
```

Comment:
The regex is a bit surprising, I'd do this if it is possible that there is file like `foo.bar.asciidoc`.

```suggestion
            ) | sed 's,.*/,,; s/\.[^.]*$//'
```

Reply:
I was going to leave the regex the same since I didn't strictly-speaking need to change it, but yeah, I don't know why it's worrying about slashes when all the slashes should have been removed by the previous replacement. Your suggestion makes more sense.